### ISSUE TITLE
Second key for Brave Payments is not displayed on the UI after recovering wallet

### ISSUE THREAD

**Issue by LaurenWags** [brave/browser-laptop#9740]
## Test plan

See below

----

- Did you search for similar issues before submitting this one?
Yes

- Describe the issue you encountered:
Second key for Brave Wallet is not displayed on the UI after recovery. Does not matter if you import keys or copy/past them into fields manually.

- Platform (Win7, 8, 10? macOS? Linux distro?):
MacOS

- Brave Version (revision SHA):
Brave	0.17.11
rev	bc6d5b8
Muon	4.1.4

- Steps to reproduce:
    1. Enable Payments in Preferences.
    2. Click on the gear icon to open Advanced Settings.
    3. Select Recover your wallet.
    4. Import recovery keys from a previously saved file.
    5. You will get a successful message. Click OK.
    6. Click on gear icon again.
    7. Click on backup your wallet.
    8. Key 2 is not displayed.
    
- Actual result:
Key 2 is not displayed.

- Expected result:
Both keys should be displayed

- Will the steps above reproduce in a fresh profile? If not what other info can be added?
Yes

- Is this an issue in the currently released version?
Yes

- Can this issue be consistently reproduced?
Yes

- Extra QA steps:
    1. Repeat steps above except manually copy/paste keys from file instead of importing. Key 2 is still not displayed on the UI.
    2. 
    3.

- Screenshot if needed:

- Any related issues:

**Comment by bsclifton** [comment]
Per https://github.com/brave/browser-laptop/issues/10442#issuecomment-322795636, @luixxiul reports that clicking the copy buttons give an error to console:
```
An uncaught exception occurred in the main process Uncaught Exception:
TypeError: Error processing argument at index 0, conversion failure from null
    at EventEmitter.ipcMain.on (/usr/lib/brave/resources/app.asar/app/index.js:255:26)
    at emitTwo (events.js:106:13)
    at EventEmitter.emit (events.js:194:7)
    at WebContents.<anonymous> (/usr/lib/brave/resources/electron.asar/browser/api/web-contents.js:182:13)
    at emitTwo (events.js:106:13)
    at WebContents.emit (events.js:194:7)
```

**Comment by zentagonist** [comment]
Revisit after #10953 is merged

**Comment by NejcZdovc** [comment]
This was broken in client library, so I upstreamed the fix

https://github.com/brave-intl/bat-client/pull/6
https://github.com/brave/ledger-client/pull/23

**Comment by LaurenWags** [comment]
Still experiencing this issue on 0.19.28.

**Comment by NejcZdovc** [comment]
we upgraded libraries that have this fix in #11231
